place_tracker/.gitignore

```diff
@@ -0,0 +1,71 @@
+# Miscellaneous
```

Comment:
@RedBrogdon 
Is there anything in .gitignore that I need to modify to put this in flutter/samples?